eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/forkchoice/ForkChoiceTestExecutor.java

```diff
@@ -47,11 +49,26 @@
   public static final ImmutableMap<String, TestExecutor> FORK_CHOICE_TEST_TYPES =
       ImmutableMap.<String, TestExecutor>builder()
           .put("fork_choice/get_head", new ForkChoiceTestExecutor())
-          .put("fork_choice/on_block", TestExecutor.IGNORE_TESTS)
+          .put(
+              "fork_choice/on_block",
+              new ForkChoiceTestExecutor(
+                  "new_finalized_slot_is_justified_checkpoint_ancestor",
+                  "new_justified_is_later_than_store_justified"))
           .build();
 
+  private List<?> testsToSkip;
+
+  public ForkChoiceTestExecutor(String... testsToSkip) {
+    this.testsToSkip = Arrays.asList(testsToSkip);
```

Comment:
nit: Slightly shorter is `List.of` but same result either way.

Reply:
and avoids `Arrays` import